### PR TITLE
chore: tweak contributing readme

### PR DESCRIPTION
**Pull request by jasonkuhrt**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,13 +13,37 @@ pnpm run setup
...
+1. `docker-compose up -d`
+
+Start the tests
+
+1. `cd src/packages/cli`
+2. `pnpm run test:postgresql && pnpm run test:mysql && pnpm run test:mariadb`
+
+Notes:
+
+- To update the snaphot add the following env var `SNAPSHOT_UPDATE=1`
```
Comment (Jolg42) [inline CONTRIBUTING.md:45]:
```suggestion
- To update the snapshots add the following env var `SNAPSHOT_UPDATE=1`
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -13,13 +13,37 @@ pnpm run setup
...
+
+Start the test databases (see [readme](./src/docker) for various ways to run these)
+
+1. `cd src/docker`
+1. `docker-compose up -d`
+
+Start the tests
+
+1. `cd src/packages/cli`
+2. `pnpm run test:postgresql && pnpm run test:mysql && pnpm run test:mariadb`
```
Comment (Jolg42) [inline CONTRIBUTING.md:41]:
```suggestion
2. `pnpm run test`
```

You missed sqlite here and stuff, all are inside the generic "test"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -13,13 +13,37 @@ pnpm run setup
 
 Note for Windows: Use the latest version of [Git Bash](https://gitforwindows.org/)
 
+### General prerequisites
+
+1. Install [`pnpm`](https://pnpm.js.org/) (for installing npm dependencies)
+1. Install [`docker`](https://www.docker.com/products/docker-desktop) (for managing test databases)
```
Comment (Jolg42) [inline CONTRIBUTING.md:19]: We could add install `ts-node`I guess

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -13,13 +13,37 @@ pnpm run setup
...
-3. `ts-node fixtures/generate.ts ./fixtures/blog/ --skip-transpile`
-4. `cd fixtures/blog`
-5. `prisma migrate save --name init --experimental && prisma migrate up --experimental`
-6. `ts-node main.ts`
+1. `cd src/packages/client`
+1. `ts-node fixtures/generate.ts ./fixtures/blog/ --skip-transpile`
+1. `cd fixtures/blog`
+1. `export DB_URL=YOUR_POSTGRES_DB_URL`  
+   For this step you might find our [docker-compose setup](./src/docker) helpful
+1. `npx prisma migrate save --name init --experimental && prisma migrate up --experimental`
```
Comment (Jolg42) [inline CONTRIBUTING.md:28]:
```suggestion
1. `npx @prisma/cli migrate save --create-db --name init --experimental && npx @prisma/cli migrate up --experimental`
```

So it works even if you don't have a Prisma CLI isntalled globally

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -13,13 +13,37 @@ pnpm run setup
 
 Note for Windows: Use the latest version of [Git Bash](https://gitforwindows.org/)
 
+### General prerequisites
+
+1. Install [`pnpm`](https://pnpm.js.org/) (for installing npm dependencies)
```
Comment (Jolg42) [inline CONTRIBUTING.md:18]: Actually we require a specific version of pnpm
```
npm i -g pnpm@5.1.7
```

File: CONTRIBUTING.md (hunk 2 above)
Comment (jasonkuhrt) [reply CONTRIBUTING.md:41]: nice

File: CONTRIBUTING.md (hunk 3 above)
Comment (jasonkuhrt) [reply CONTRIBUTING.md:19]: Maybe missed something but didn't see any raw ts-node invocation use-cases for the project?

File: CONTRIBUTING.md (hunk 4 above)
Comment (jasonkuhrt) [reply CONTRIBUTING.md:28]: Ah yep!

File: CONTRIBUTING.md (hunk 5 above)
Comment (jasonkuhrt) [inline CONTRIBUTING.md:18]:
```suggestion
1. Install [`pnpm@5.1.7`](https://pnpm.js.org/) (for installing npm dependencies)
```

File: CONTRIBUTING.md (hunk 3 above)
Comment (Jolg42) [reply CONTRIBUTING.md:19]: Line 19 below we have `1. `ts-node fixtures/generate.ts ./fixtures/blog/ --skip-transpile``

**Comment by Jolg42** [comment]
Thanks so much for the PR 💚

File: CONTRIBUTING.md (hunk 3 above)
Comment (jasonkuhrt) [inline CONTRIBUTING.md:19]:
```suggestion
1. Install [`docker`](https://www.docker.com/products/docker-desktop) (for managing test databases)
1. Install [`ts-node`](https://github.com/TypeStrong/ts-node) (for running Node scripts written in TypeScript)
```

**Pull request merged**